pom.xml

```diff
@@ -48,16 +48,17 @@
          https://github.com/confluentinc/common (see common's pom.xml for kafka.version).
     -->
     <properties>
-        <algebird.version>0.13.4</algebird.version>
+        <algebird.version>0.13.7</algebird.version>
         <avro.version>1.9.1</avro.version>
-        <chill.version>0.9.2</chill.version>
+        <chill.version>0.9.5</chill.version>
         <docker.skip-build>false</docker.skip-build>
         <docker.skip-test>false</docker.skip-test>
         <java.version>1.8</java.version>
         <licenses.version>6.0.0-SNAPSHOT</licenses.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <scala.version>${kafka.scala.version}.8</scala.version>
-        <scalatest.version>3.0.5</scalatest.version>
+        <kafka.scala.version>2.13</kafka.scala.version>
```

Comment:
Why can't we bump `${kafka.scala.version}` instead of hard coding it?

Reply:
I think it's actually clearer to not inherit this value from dependencies, since if in the future it fails again we know what is causing the inconsistency more easily.

Reply:
Agree. This addresses the fear I expressed in https://github.com/confluentinc/kafka-streams-examples/pull/338#discussion_r433317436